AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -3195,4 +3195,6 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 ! Detect Advanced Protection of AdGuard for iOS
 !+ PLATFORM(ios) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app#$#.hello_from_adguard_advanced_protection_ios { display: none !important; }
-!
+! Annoying elements, related to social networks, which we do not block in the main Social filter
+inkorr.com##.sub-tg-wrapper
```

Comment:
Better to change it to smth like this.
```adblock
inkorr.com##.content--main  div.sub-tg-wrapper
```
Add just the rule without the comment here)
<details><summary>Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/33488229/51d3e179-92c4-40e8-a7dc-2b0f0f3a39be)

</details><br/>